nipype/interfaces/base.py

```diff
@@ -1452,12 +1448,28 @@ def _process(drain=0):
             result['merged'] = result['stdout']
             result['stdout'] = []
     else:
-        proc.communicate()  # Discard stdout and stderr
+        stdout, stderr = proc.communicate()
```

Comment:
Maybe make these `stdoutstr`, `stdinstr`, to be consistent with `if output.startswith('file')`, so that the same `del` statement will work in all cases.

Alternately, it might be simplest to just run `gc.collect()` in `CommandLine._run_interface`, after this function is called and everything is allowed to go out of scope.